From 1fadd50f154ea365d60663e1a41ef85a9e981b4d Mon Sep 17 00:00:00 2001
From: Sai Vemuri <svemuri@redhat.com>
Date: Tue, 2 Dec 2014 22:33:55 -0500
Subject: [redhat] configs: Disable Chelsio FCOE driver

Message-id: <1417559635-1104-3-git-send-email-svemuri@redhat.com>
Patchwork-id: 101101
O-Subject: [RHEL7.1 PATCH BZ 1169941 2/2] Disable Chelsio FCOE driver
Bugzilla: 1169941
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Nikolay Aleksandrov <nikolay@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

The Chelsio FCoE offload driver (csiostor) is leaving the cards in hung
state. This patch disables it till the fixes are available.

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/CONFIG_SCSI_CHELSIO_FCOE b/redhat/configs/generic/CONFIG_SCSI_CHELSIO_FCOE
index 8acf63e..ad8110d 100644
--- a/redhat/configs/generic/CONFIG_SCSI_CHELSIO_FCOE
+++ b/redhat/configs/generic/CONFIG_SCSI_CHELSIO_FCOE
@@ -1 +1 @@
-CONFIG_SCSI_CHELSIO_FCOE=m
+# CONFIG_SCSI_CHELSIO_FCOE is not set
-- 
1.7.1